charts/tikv-importer/templates/_helpers.tpl

```diff
@@ -14,6 +14,12 @@ config-file: |-
     {{- if .Values.config }}
 {{ .Values.config | indent 2 }}
     {{- end -}}
+    {{- if and .Values.tlsCluster .Values.tlsCluster.enabled }}
+  [security]
+  ca-path="/etc/lib/importer-tls/ca.crt"
```

Comment:
ditto.

Reply:
fixed in https://github.com/pingcap/tidb-operator/pull/3598/commits/704b30b0f4c26e52ff41e1cf023b9ea1bc611ef1.